boards/arm/npcx7m6fb_evb/doc/index.rst

```diff
@@ -79,10 +79,20 @@ JTAG only sessions.
 Flashing
 ========
 
-Build application as usual for the ``npcx7m6fb_evb`` board, and flash
-using Servo V2, μServo, or Servo V4 (CCD). See the
+If the correct headers are installed, this board supports both J-TAG and also
```

Comment:
headers? correct drivers/libraries/tools? headers sounds like C headers, to me at least :-)

Reply:
The headers mean the corresponding slots on the [boards](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/npcx9m6f_evb/doc/npcx9m6f_evb.jpg) I will replace it with `slots`.

Reply:
ohhh! "pin headers" :-)

Reply:
guess "IDC header" if you want to be pedantic about naming :-D

Reply:
Done.